packages/shared_preferences/shared_preferences_android/android/src/main/java/io/flutter/plugins/sharedpreferences/MethodCallHandlerImpl.java

```diff
@@ -1,224 +0,0 @@
-// Copyright 2013 The Flutter Authors. All rights reserved.
-// Use of this source code is governed by a BSD-style license that can be
-// found in the LICENSE file.
-
```

Comment:
Please don't move all of the existing code to a different file in this PR; it makes it extremely difficult to review the actual changes. The code that was in this file should stay there for now, with just the minimal changes to adapt it to Pigeon.

Reply:
Okay, I will do that, sorry it's my first time. I will try to make the reviewing process as smooth as possible.